<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicePage</class>
 <widget class="QWidget" name="DevicePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="CustomImageLabel" name="buttonVideo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonTrigger">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonGsensor">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>90</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonBuzzer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>360</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Input</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTriggerText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Input</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGsensorText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>220</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Impact
Sensitivity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBuzzerText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Buzzer Output</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFormatText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>SSD Format</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCloseText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>490</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonFormat">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomImageLabel</class>
   <extends>QLabel</extends>
   <header>ui/customwidget/customimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
